package.json

```diff
@@ -16,7 +16,7 @@
         "Maximo",
         "IBM",
         "ArcGIS",
-        "GIS"
+        "GIS"        
```

Comment:
Extra spaces added for some reason.

Reply:
Removed (and updated Geocortex to VertiGIS Studio)